dapps/evancore.vue.libs/src/components/forms/form/form.ts

```diff
@@ -220,22 +220,32 @@ export default class EvanFormComponent extends mixins(EvanComponent) {
 
     // return directly specified translation
     let returnTranslation = type !== 'hint';
-    if (control.uiSpecs && control.uiSpecs.attr &&
-        control.uiSpecs.attr.hasOwnProperty(type)) {
+    if (control.uiSpecs &&
+        (control.uiSpecs.hasOwnProperty(type) ||
+          (control.uiSpecs.attr && control.uiSpecs.attr.hasOwnProperty(type)))
```

Comment:
I would consider refactoring into a small check function when you need 3 lines in the if-bracket